.gitlab-ci/template/include/preferred-wpe-2_32.conf

```diff
@@ -1,6 +1,6 @@
 PREFERRED_PROVIDER_virtual/wpebackend = "wpebackend-fdo"
 PREFERRED_PROVIDER_virtual/libwpe = "libwpe"
-PREFERRED_VERSION_cog = "0.7.%"
-PREFERRED_VERSION_wpewebkit = "2.28.%"
-PREFERRED_VERSION_wpebackend-fdo = "1.6.%"
+PREFERRED_VERSION_cog = "0.8.%"
+PREFERRED_VERSION_wpewebkit = "2.32.0%"
+PREFERRED_VERSION_wpebackend-fdo = "1.8.%"
```

Comment:
WPEBackend-fdo 1.10.0 will be released in the coming days, maybe you want to wait until then and depend on 1.10.0 instead.

Reply:
It is a possibility but having in account WPE is [compatible with 1.8](https://wpewebkit.org/release/schedule/) I don't see this as a blocker.